source/modules/sidebar/sidebar.pug

```diff
@@ -0,0 +1,29 @@
+mixin sidebar()
+  .sidebar
+    h2.sidebar__user-avatar
+        a(href="#") User avatar image
+    p.sidebar__username Emily Doe
+    ul.nav.nav-stacked.sidebar__nav
+      li.nav-item.active
+        a.nav-link(href='#')
+          i.anticon.icon-user
+          span.text User profile
+      li.nav-item
```

Comment:
please using data json. after that using loop to print it

Reply:
I will update this with Data.json in page p002-p003-user-and-settings (next pull)

Reply:
please add for this pull. pack it !